### ISSUE TITLE
can't work with -DTEST()="hello"

### ISSUE THREAD

**Issue by johnzeng** [justmao945/vim-clang#110]
set up a `.clang` file with the following content:

```
-D TEST(A)="hello"
```

and then you will get error because you didn't escape `()` in the command....

**Comment by justmao945** [comment]
-D 'TEST(A)="hello"'

**Comment by johnzeng** [comment]
I don't think this is a good enough solution for a plugin... but ... this plugin is just passing parameters to a shell so I think there is no such thing called 'perfect solution' so I give up. Do you mind to update the readme file to let other people knows this inconvenient?
